sys/include/dbgpin_params.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2020 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_dbgpin
+ *
+ * @{
+ * @file
+ * @brief       Default parameters for the dbgpin helper module
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ */
+
+#ifndef DBGPIN_PARAMS_H
+#define DBGPIN_PARAMS_H
+
+#include "periph/gpio.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#ifndef DBGPIN_PARAMS
+#define DBGPIN_PARAMS   GPIO_PIN(0,0)
+#endif
```

Comment:
This default might conflict with some boards, at least with `boards/slstk340*`

Reply:
Yes, thats the problem about a default pin, it will never fit all platforms... So far this simply copies the behavior from any driver _params.h file: simply start with pin0,0 as default value.

But thinking about this, I tend to remove the default pin in the first place and simply output an error in case the user did not specify any DBGPIN_PARAMS.